src/components/VisuallyHidden/VisuallyHidden.tsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * Used to make tooltips accessible. This hidden div contains a description of the
+ * tooltip's content.
+ * The div's ID (elemId) is referenced by the tooltip's aria-describedby attribute.
+ * https://inclusive-components.design/tooltips-toggletips/
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+
+export const HiddenDiv = styled.div`
+  clip-path: inset(100%);
+  clip: rect(1px, 1px, 1px, 1px);
+  height: 1px;
+  overflow: hidden;
+  position: absolute;
+  white-space: nowrap;
+  width: 1px;
+`;
+
+const VisuallyHiddenDiv: React.FC<{
+  content: React.ReactNode;
+  elemId: string;
+}> = ({ content, elemId }) => {
```

Comment:
I don't think we need the `VisuallyHiddenDiv` component, we can just pass the id, children and role to the `HiddenDiv` component directly:

```tsx
<HiddenDiv id={idForAccessability} role="tooltip">
  {props.title}
</HiddenDiv>
```

Reply:
per our chat, keeping it as is